spec/acceptance/firewall_attributes_exceptions_spec.rb

```diff
@@ -405,7 +405,7 @@ class { '::firewall': }
 
   # iptables version 1.3.5 is not suppored by the ip6tables provider
   # iptables version 1.4.7 fails for multiple hl entries
-  if default['platform'] !~ %r{(el-5|el-6|sles-10|sles-11)}
+  describe 'testing ipv6', unless: (os[:family] == 'redhat' && ['5', '6'].include?(os[:release][0])) || (os[:family] == 'sles') do
```

Comment:
You can just do "os[:release].start_with?('5', '6')".

Reply:
nice catch, i wasnt aware that start_with accepted multiple prefixes.

![image](https://user-images.githubusercontent.com/10402706/53821711-3cc0e780-3f66-11e9-92bf-efe24f38d7de.png)